src/Client.php

```diff
@@ -64,6 +87,11 @@ public function __construct(string $accessToken, GuzzleClient $client = null, in
      * If the source path is a folder all its contents will be copied.
      *
      * @link https://www.dropbox.com/developers/documentation/http/documentation#files-copy_v2
+     *
+     * @param string $fromPath
```

Comment:
Do not add docs block where everything is already type hinted.

Reply:
Removed